app/models/manageiq/providers/redfish/physical_infra_manager/operations/firmware.rb

```diff
@@ -0,0 +1,55 @@
+module ManageIQ::Providers::Redfish
+  module PhysicalInfraManager::Operations::Firmware
+    def update_firmware_async(firmware_binary, servers)
+      validate_update_firmware(firmware_binary, servers)
+
+      with_provider_connection do |client|
+        update_service = client.UpdateService
+        raise MiqException::MiqFirmwareUpdateError, 'UpdateService is not enabled' unless update_service
+
+        protocol, url = compatible_firmware_url(update_service, firmware_binary)
+
+        response = update_service.post(
+          :path    => update_service.Actions['#UpdateService.SimpleUpdate'].target,
+          :payload => {
+            :ImageURI         => url,
+            :TransferProtocol => protocol,
+            :Targets          => servers.map(&:ems_ref),
+          }
+        )
+
+        if response.status.to_i != 200
```

Comment:
If the Redfish service started the firmware update operation, it will return status 202, not 200.

And a nitpick: all of the  `RedfishClient::Response`s that are returned by the Redfish client are guaranteed to return integer status, so no need to call `to_i` here.